### PR TITLE
feat: support fish_indent

### PR DESCRIPTION
**Pull request by hougesen**

### CODE REVIEW

File: src/languages/shell.rs [hunk 1]
```diff
@@ -42,6 +48,7 @@ impl core::fmt::Display for Shell {
         match self {
             Self::Shfmt => write!(f, "shfmt"),
             Self::Beautysh => write!(f, "beautysh"),
+            Self::FishIndent => write!(f, "fish_indent"),
```
Comment (ccoVeille) [inline src/languages/shell.rs:51]: I'm unsure I understand the code, but 

I would expect fish_ident to be used only with fish  `markdown` identifier

Fish shell is not POSIX, calling it on bash, sh, or zsh may produce invalid code

Comment (ccoVeille) [reply src/languages/shell.rs:51]: But I'm unsure there is a standardized fish identifier

https://github.com/github-linguist/linguist/blob/master/lib%2Flinguist%2Flanguages.yml

Comment (ccoVeille) [reply src/languages/shell.rs:51]: It's used pretty often anyway

https://github.com/search?q=%22%60%60%60fish%22+language%3Amd&type=code

Comment (hougesen) [reply src/languages/shell.rs:51]: Aaah, I am not a fish user so hadn't really looked into it, just thought it was some minor differences 😅

Comment (ccoVeille) [reply src/languages/shell.rs:51]: I would say it may happen.

Comment (ccoVeille) [reply src/languages/shell.rs:51]: Thanks

**Pull request merged**